README.md

```diff
@@ -52,8 +52,17 @@ DDD is a software design approach that is against the idea of having a single un
 
 What this means for a React application, is that instead of using one global state with, for example Redux, we will have a React context for each of our entities that will store all the data and offer all the methods related to that domain.
 
+- The domain folder contains all the entities and repositories. (Types, Interfaces, etc.)
+- The infrastructure folder contains the implementations of the interfaces. (API calls, etc.)
+- The application folder contains the services that use the infrastructure to implement the business logic.
```

Comment:
we don't (and probably won't) have an applications folder, only app folder, but that's for next, not services